src/types/proxy.ts

```diff
@@ -0,0 +1,24 @@
+import { Context, SupportedEvents, SupportedEventsU } from "./context";
+
+export type CallbackResult = { status: 200 | 201 | 204 | 404 | 500; reason: string; content?: string | Record<string, unknown> };
```

Comment:
I saw a nice object in another plug-in where I think you can import all the http status codes and then select. 

Maybe it's more convenient with more type information to do this. If not, no big deal.

Reply:
I remember that too, it may have been me but I can't remember and cannot find the package. I'm sure it was GitHub related but no luck with it